introduction.md

```diff
@@ -1,3 +1,5 @@
 hello, but it looks to be empty, or just too short. We really want to know something about you! Make sure to include a short sentance about yourself.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?